sdk/python/test_data/pipelines/if_elif_else.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from kfp import compiler
+from kfp import dsl
+
+
+@dsl.component
+def flip_coin() -> str:
```

Comment:
nit: maybe rename this function? "flipping a coin" typically expects a binary result, and the addition of `draw` makes the example a bit odd.

Reply:
Updated.